src/ig/IGExporter.ts

```diff
@@ -294,9 +294,10 @@ export class IGExporter {
     examples.forEach(example => {
```

Comment:
Thank you GitHub for not letting me comment on the line I want, but 293 is 
`const examples = sortBy(this.pkg.instances, instance => instance.id ?? instance.instanceName);`
This should be changed to use `instance._instanceMeta.instanceName` right?

Reply:
Yes.  Not sure how that one escaped my global search!